pkg/controllers/subnetset/subnetbinding_handler.go

```diff
@@ -0,0 +1,98 @@
+package subnetset
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/util/workqueue"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	"github.com/vmware-tanzu/nsx-operator/pkg/apis/vpc/v1alpha1"
+)
+
+func requeueSubnetSetBySubnetBindingCreate(ctx context.Context, c client.Client, obj client.Object, q workqueue.TypedRateLimitingInterface[reconcile.Request]) {
+	bindingMap := obj.(*v1alpha1.SubnetConnectionBindingMap)
+	if bindingMap.Spec.TargetSubnetSetName != "" {
+		enqueue(ctx, c, bindingMap.Namespace, bindingMap.Spec.TargetSubnetSetName, q)
+	}
+}
+
+func requeueSubnetSetBySubnetBindingUpdate(ctx context.Context, c client.Client, objOld, objNew client.Object, q workqueue.TypedRateLimitingInterface[reconcile.Request]) {
+	newBM := objNew.(*v1alpha1.SubnetConnectionBindingMap)
+	oldBM := objOld.(*v1alpha1.SubnetConnectionBindingMap)
+
+	if newBM.Spec.TargetSubnetSetName == oldBM.Spec.TargetSubnetSetName {
+		return
+	}
+
+	if newBM.Spec.TargetSubnetSetName != "" {
+		if err := enqueue(ctx, c, newBM.Namespace, newBM.Spec.TargetSubnetSetName, q); err != nil {
+			return
```

Comment:
if there is error when enqueuing the new targetSubnetSet, the old targetSubnetSet will not be enqueued, will this cause issue?